.ruby-version

```diff
@@ -1 +1 @@
-2.4.0
+2.2.10
```

Comment:
? Assume this is unrelated? Should we remove the constraint entirely or is this what's used in Production?